### PR TITLE
en: Update info on EKS and dashboard access

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added, or deleted? (Required)

- Add more details about accessing TiDB with an nlb with internal scheme
- Link to info about using TiDB Dashboard
- Update info about TiDB Dashboard with Kubernetes

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- https://github.com/pingcap/tidb-operator/pull/3974
- https://github.com/pingcap/tidb-operator/issues/3941

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM

File: en/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -242,6 +242,10 @@ Allow the bastion host to access the Internet. Select the correct key pair so th
 
 After the bastion host is created, you can connect to the bastion host via SSH and access the TiDB cluster via the MySQL client.
 
+> **Note:**
+>
+> The configuration in `tidb-cluster.yaml` setups the loadbalancer for TiDB with the "internal" scheme. This means that it is only accessible from within the VPC and not externally. To access TiDB over the MySQL protocol you need to use a bastion host or use `kubectl port-forward`. If you want to expose TiDB over the internet and if you are aware of the risks of doing this you can change the scheme for the loadbalancer from "internal" to "internet-facing" and re-deploy the loadbalancing service.
```
Comment (qiancai) [inline en/deploy-on-aws-eks.md:247]:
```suggestion
> By default, the configuration in `tidb-cluster.yaml` sets up the LoadBalancer for TiDB with the "internal" scheme. This means that the LoadBalancer is only accessible within the VPC, not externally. To access TiDB over the MySQL protocol, you need to use a bastion host or use `kubectl port-forward`. If you want to expose TiDB over the internet and if you are aware of the risks of doing this, you can change the scheme for the LoadBalancer from "internal" to "internet-facing" in the `tidb-cluster.yaml` file, and then re-deploy the loadbalancing service.
```

File: en/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -242,6 +242,10 @@ Allow the bastion host to access the Internet. Select the correct key pair so th
 
 After the bastion host is created, you can connect to the bastion host via SSH and access the TiDB cluster via the MySQL client.
 
+> **Note:**
+>
+> By default, the configuration in `tidb-cluster.yaml` sets up the LoadBalancer for TiDB with the "internal" scheme. This means that the LoadBalancer is only accessible within the VPC, not externally. To access TiDB over the MySQL protocol, you need to use a bastion host or use `kubectl port-forward`. If you want to expose TiDB over the internet and if you are aware of the risks of doing this, you can change the scheme for the LoadBalancer from "internal" to "internet-facing" in the `tidb-cluster.yaml` file, and then re-deploy the loadbalancing service.
```
Comment (qiancai) [inline en/deploy-on-aws-eks.md:247]: According to @DanielZhangQD, users are not recommended to update "internal" to "internet-facing" directly in the tidb-cluster.yaml file after the deployment. Could we move the note to the Deploy steps, just after the "Refer to configure the TiDB cluster to further customize and configure the CR before applying." sentence? Then users will know this "internal" scheme and can determine whether to change it to "external" during the configuration.

File: en/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -179,6 +179,10 @@ curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/
 
 Refer to [configure the TiDB cluster](configure-a-tidb-cluster.md) to further customize and configure the CR before applying.
 
+> **Note:**
+>
+> By default, the configuration in `tidb-cluster.yaml` sets up the LoadBalancer for TiDB with the "internal" scheme. This means that the LoadBalancer is only accessible within the VPC, not externally. To access TiDB over the MySQL protocol, you need to use a bastion host or use `kubectl port-forward`. If you want to expose TiDB over the internet and if you are aware of the risks of doing this, you can change the scheme for the LoadBalancer from "internal" to "internet-facing" in the `tidb-cluster.yaml` file, and then re-deploy the loadbalancing service.
```
Comment (DanielZhangQD) [inline en/deploy-on-aws-eks.md:184]:
```suggestion
> By default, the configuration in `tidb-cluster.yaml` sets up the LoadBalancer for TiDB with the "internal" scheme. This means that the LoadBalancer is only accessible within the VPC, not externally. To access TiDB over the MySQL protocol, you need to use a bastion host or use `kubectl port-forward`. If you want to expose TiDB over the internet and if you are aware of the risks of doing this, you can change the scheme for the LoadBalancer from "internal" to "internet-facing" in the `tidb-cluster.yaml` file.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**